File: platform/ios/demo/Examples/ObjectiveC/DefaultStylesExample.m
#import "DefaultStylesExample.h"
@import Mapbox;

NSString *const MBXExampleDefaultStyles = @"DefaultStylesExample";

@implementation DefaultStylesExample

- (void)viewDidLoad {
    [super viewDidLoad];

    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds
                                                   styleURL:[MGLStyle outdoorsStyleURLWithVersion:9]];

    // Tint the ℹ️ button and the user location annotation.
    mapView.tintColor = [UIColor darkGrayColor];

    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;

    // Set the map’s center coordinate and zoom level.
    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(51.50713, -0.10957)
                       zoomLevel:13
                        animated:NO];

    [self.view addSubview:mapView];
}

@end

File: platform/ios/demo/Examples/Swift/DefaultStylesExample.swift
import Mapbox
    
@objc(DefaultStylesExample_Swift)

class DefaultStylesExample_Swift: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let mapView = MGLMapView(frame: view.bounds,
                                 styleURL: MGLStyle.outdoorsStyleURL(withVersion: 9))
        
        // Tint the ℹ️ button and the user location annotation.
        mapView.tintColor = .darkGray
        
        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
        
        // Set the map’s center coordinate and zoom level.
        mapView.setCenter(CLLocationCoordinate2D(latitude: 51.50713,
            longitude: -0.10957),
            zoomLevel: 13, animated: false)
        view.addSubview(mapView)
    }
}
